src/settings/JobProfiles/ViewJobProfile.js

```diff
@@ -148,21 +141,12 @@ export class ViewJobProfile extends Component {
         id: PropTypes.string,
       }).isRequired,
     }).isRequired,
-    location: PropTypes.oneOfType([
-      PropTypes.shape({
-        search: PropTypes.string.isRequired,
-        pathname: PropTypes.string.isRequired,
-      }).isRequired,
-      PropTypes.string.isRequired,
-    ]),
     tagsEnabled: PropTypes.bool,
     onClose: PropTypes.func.isRequired,
     onDelete: PropTypes.func.isRequired,
     paneId: PropTypes.string, // eslint-disable-line
     ENTITY_KEY: PropTypes.string, // eslint-disable-line
     actionMenuItems: PropTypes.arrayOf(PropTypes.string), // eslint-disable-line
-    withEditRecordButton: PropTypes.bool,
-    withRunRecordButton: PropTypes.bool,
```

Comment:
It seems like these props are unused. Remove these props from the `UploadingJobsDisplay.js` file, please.

Reply:
Done!